File: src/main/swift/Sources/swift/fr/jhelp/kotlinLightSamples/blueetooth/BluetoothInterfaceDummy.swift

class BluetoothInterfaceDummy : BluetoothInterface {

    func initialize()
    {
        print("TODO : Inject BluetoothInterface")
    }

    func startScan(_ scanReceiver: @escaping  (BluetoothDevice) -> Void)
    {
        print("TODO : Inject BluetoothInterface")
    }

    func stoScan()
    {
        print("TODO : Inject BluetoothInterface")
    }

    func connect(_ device: BluetoothDevice)
    {
        print("TODO : Inject BluetoothInterface")
    }
}

File: src/main/kotlin/fr/jhelp/kotlinLightSamples/blueetooth/BluetoothInterfaceDummy.kt
package fr.jhelp.kotlinLightSamples.blueetooth

class BluetoothInterfaceDummy : BluetoothInterface
{
    override fun initialize()
    {
        println("TODO : Inject BluetoothInterface")
    }

    override fun startScan(scanReceiver: (BluetoothDevice) -> Unit)
    {
        println("TODO : Inject BluetoothInterface")
    }

    override fun stoScan()
    {
        println("TODO : Inject BluetoothInterface")
    }

    override fun connect(device: BluetoothDevice)
    {
        println("TODO : Inject BluetoothInterface")
    }
}